### PR TITLE
Modified design of FAQ and created separate file for inline css

### PR DESCRIPTION
**Pull request by sakshikhachane**
## Issue that this pull request solves

 Closes: #102 

### Brief description of what is fixed or changed
The design of the FAQ page is changed and a separate file is created for all the inline CSS which was present in the FAQ page.

## Types of changes

_Put an `x` in the boxes that apply_

- [`x`] Bugfix (non-breaking change which fixes an issue)
- [`x`] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [`x`] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [`x`] My changes generate no new warnings
- [ 'x'] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of a change in the user interface
HITK Tech Community - Google Chrome 29-12-2020 21_55_39
HITK Tech Community - Google Chrome 29-12-2020 21_55_35
HITK Tech Community - Google Chrome 29-12-2020 21_55_25

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: frontend/src/components/faq/faqs.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+}
+.MuiPaper-root.Mui-expanded, .MuiPaper-root::after{
+    box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.2), 0 4px 5px 0 rgba(0, 0, 0, 0.19);
+}
+
+#panel1bh-header.Mui-expanded, #panel2bh-header.Mui-expanded, #panel3bh-header.Mui-expanded, #panel4bh-header.Mui-expanded{
+    background-color: #eff8ff   ;
+   
+}
+.container{
```
Comment (Kajol-Kumari) [inline frontend/src/components/faq/faqs.css:66]: This class is conflicting somehow. As it's changing the styles on `about-us` and `motive` section of the home page. Please fix it.

Screenshot from 2020-12-30 13-09-39

File: frontend/src/components/faq/faqs.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+
+
+.faq-block:hover, #panel1bh-header:hover, #panel2bh-header:hover,  #panel3bh-header:hover,  #panel4bh-header:hover{
+    padding-left: 15px;
+    background-color: #cae4db;
+    
+}
```
Comment (Kajol-Kumari) [inline frontend/src/components/faq/faqs.css:78]: leave one extra blank line at the end of the file

File: frontend/src/components/faq/faqs.css [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+#panel1bh-header.Mui-expanded, #panel2bh-header.Mui-expanded, #panel3bh-header.Mui-expanded, #panel4bh-header.Mui-expanded{
+    background-color: #eff8ff   ;
+   
+}
+.container{
+    margin-top: 50px;
+    background-color: #d9ecf2;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
```
Comment (Kajol-Kumari) [inline frontend/src/components/faq/faqs.css:71]: just leave single line space between two css classes, remove the rest blank lines

File: frontend/src/components/faq/faqs.css [hunk 4]
```diff
@@ -0,0 +1,78 @@
+.faq-image{
+    width:100%;
+    height:90vh;
+    background: rgba(0, 0, 0, 0.3);
+}
+
+
```
Comment (Kajol-Kumari) [inline frontend/src/components/faq/faqs.css:7]: leave only single blank space

File: frontend/src/components/faq/faqs.css [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+.faqs-top-section{
+    position:relative;
+}
+.faqs-top-section h1{
```
Comment (Kajol-Kumari) [inline frontend/src/components/faq/faqs.css:11]: leave a single blank line here before this class

File: frontend/src/components/faq/faqs.css [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+#panel1bh-content,#panel2bh-content, #panel3bh-content, #panel4bh-content{
+    background-color: #f1f9f9;
+}  
+
+.faq-block{
+   
+    box-shadow: 0 4px 2px 0 rgba(0, 0, 0, 0.19);
+}
+
+
```
Comment (Kajol-Kumari) [inline frontend/src/components/faq/faqs.css:53]: remove one extra blank line space here

File: frontend/src/components/faq/faqs.css [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+.faq-question{
+    font-weight: 600; 
+    font-size: 1rem; 
+}
+.MuiPaper-root.Mui-expanded, .MuiPaper-root::after{
```
Comment (Kajol-Kumari) [inline frontend/src/components/faq/faqs.css:58]: leave a single blank line here

File: frontend/src/components/faq/faqs.js [hunk 8]
```diff
@@ -97,8 +108,12 @@ export default function Faqs() {
...
                 </Accordion>
+                <br></br>
+                
             </div>
         </div>
+        </div>
+        </div>
 
     );
 }
```
Comment (Kajol-Kumari) [inline frontend/src/components/faq/faqs.js:119]: leave one extra blank line at the end of the file

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@sakshikhachane please resolve the conflicting classes and make this minor changes

**Comment by sakshikhachane** [comment]
> @sakshikhachane please resolve the conflicting classes and make this minor changes

@Kajol-Kumari Done with the changes please review.

**Review by Kajol-Kumari — APPROVED** [review]
Thanks for incorporating the suggested changes @sakshikhachane 
LGTM :+1:

**Comment by Kajol-Kumari** [comment]
@all-contributors please add @sakshikhachane for code

**Pull request merged**